### ISSUE TITLE
no scope doesn't work

### ISSUE THREAD

**Issue by Darhagonable** [CanopyTax/kremling-babel-plugin#18]
Kremling itself supports writing inline css without scoping it like this:
```
import React from 'react';
import { useCss, k } from 'kremling';

export function MyComponent() {
  useCss(css);
  return (
    <div className="custom">
      Custom
    </div>
  );
}

const css = k`
  .custom {
    background-color: red;
  }
`;
```

Using this babel plugin this is no longer possible.
I know its a very unusual case to do this but it can be a nice feature to have still.

**Comment by geoctrl** [comment]
yeah, this is a common request in the kremling ecosystem - I'm looking into what's possible with the postcss plugin 🤞

**Comment by Darhagonable** [comment]
Has there been any further development on this?
Did it seem possible or will i have to choose between sass and noscoping?
(Not meaning to rush anything, just curious of the status)

**Comment by geoctrl** [comment]
the idea was to create some new syntax that the internal postcss plugin could read that would allow an escape hatch

kinda like how css modules `:global()` works

it's still on my list to do, just haven't had much time to look into this as of late - any MR's are welcome though

(Note: the postcss plugin lives inside the `kremling-loader` repo - you'll want to start there)
https://github.com/CanopyTax/kremling-loader/blob/master/src/postcss-kremling-plugin.js

**Comment by Darhagonable** [comment]
Ohh. I thought it would be less of a hassle since regular Kremling works without setting scope (like in the top post).
So I guess the babel plugin changes the behavior making it impossible to do the same way?

**Comment by geoctrl** [comment]
finally got to it
https://github.com/CanopyTax/kremling-loader/pull/14

**Comment by geoctrl** [comment]
implemented in `kremling-babel-plugin`! #26
